scripts/changelog.ts

```diff
@@ -0,0 +1,380 @@
+import child = require("child_process");
+import fs = require("fs");
+
+import {markup} from "@romefrontend/cli-layout";
+import {Reporter} from "@romefrontend/cli-reporter";
+import {parseCommit} from "@romefrontend/commit-parser";
+import {readFileText} from "@romefrontend/fs";
+import {AbsoluteFilePath, createAbsoluteFilePath} from "@romefrontend/path";
+
+import https = require("https");
+import http = require("http");
+
+const ROOT = (() => {
+	let pickNext = false;
+
+	for (const path of createAbsoluteFilePath(__dirname).getChain()) {
+		if (pickNext) {
+			return path;
+		}
+
+		if (path.getBasename() === "scripts") {
+			pickNext = true;
+		}
+	}
+
+	throw new Error("Could not find the root");
+})();
+
+const PACKAGES = ROOT.append("packages");
```

Comment:
You should be able to just use the existing utils, right?

```suggestion
import {PACKAGES, ROOT} from "./_util";
```